requirements3.txt

```diff
@@ -1,6 +1,7 @@
 astroid==2.3.3
 backports.functools_lru_cache==1.6.1
 configparser==4.0.2
+chardet==4.0.0
```

Comment:
This was expressed as of https://github.com/kubernetes/test-infra/blob/8b71bc27720d9818c315c02e9abdc1d7b58bdcc9/kettle/BUILD.bazel#L14, but never defined anywhere in this repo, `pip3 install -r requirements3.txt` doesn't install it as a transitive dependency either. So explicitly define it here. The other dependencies under https://github.com/kubernetes/test-infra/blob/8b71bc27720d9818c315c02e9abdc1d7b58bdcc9/kettle/BUILD.bazel#L14 are all transitive dependencies so we won't need to worry about them